test/force-auto.py.in

```diff
@@ -42,11 +42,6 @@ import inspect
 import sys
 
 
-@enum.unique
-class Scope(enum.Enum):
-   STANDARD = "standard"
-   FULL = "full"
```

Comment:
I think you need to keep the scope handling here b/c the rootemu tests take forever, so we'll want to keep `-s standard rootemu` and `-s full rootemu` both available.